src/components/DropZone/DropZone.tsx

```diff
@@ -151,10 +151,10 @@ export class DropZone extends React.Component<CombinedProps, State> {
     return Object.keys(newState).length ? newState : null;
   }
 
-  private node: HTMLElement | null = null;
-  private dropNode: HTMLElement | HTMLDocument | null = null;
+  private node: React.RefObject<HTMLDivElement>;
+  private dropNode: React.RefObject<HTMLDivElement> | Document;
```

Comment:
Why did you change the type from `HTMLDocument` to `Document`?

Reply:
Thanks for pointing that out. Will change all usage of `Document` back to `HTMLDocument` 😄